test/Array/rlexe.xml

```diff
@@ -239,6 +239,18 @@
       <baseline>pop3.baseline</baseline>
     </default>
   </test>
+  <test>
+    <default>
+      <files>pop4.js</files>
+	  <compile-flags>-bgjit- -mic:1 -off:simplejit</compile-flags>
```

Comment:
Sorry my previous comment wasn't clear, these flags shouldn't be needed here, the test runner script runs everything twice with a variety of flags, it will supply -bgjit- -mic:1 and -off:simplejit in one of those runs, compile flags in the rlexe.xml files are for weird/unusual situations.

Reply:
Test (pop5.js) still fails on master (It’s working)

Reply:
For reference, see here: https://github.com/chakra-core/ChakraCore/blob/c4db12989f118d86c2fc05a4f0af6aef2e2d2e18/test/runtests.py#L725-L734 for flags used for the default test runs, everything is done twice once with `Interpreted` and once with `dynopogo` note "maxInterpretCount" is the same flag as `mic`.